aws/rust-runtime/aws-inlineable/src/s3_express.rs

```diff
@@ -766,6 +766,29 @@ pub(crate) mod runtime_plugin {
     }
 }
 
+pub(crate) mod checksum {
+    use aws_smithy_checksums::ChecksumAlgorithm;
+    use aws_smithy_types::config_bag::ConfigBag;
+
+    pub(crate) fn provide_default_checksum_algorithm(
+        original_checksum: Option<ChecksumAlgorithm>,
+        cfg: &ConfigBag,
+    ) -> Option<ChecksumAlgorithm> {
```

Comment:
Nit: have this return DefaultRequestChecksumOverride directly so that it doesn't have to be wrapped at the call site.

Reply:
Updated in 3c3843f